src/index.html

```diff
@@ -47,13 +47,19 @@
                 });
             }
 
+            function on_select_pano(panoData) {
+                console.log(panoData);
+            }
+
 
             window.onload = function () {
                 PlantingInstance = new window.Planting( {
                     container:document.querySelector('.viewport'),
                     manifestoUrl: '/manifesto.json',
                     googleApiKey: 'AIzaSyD9fmhpMCKGM6BCMtsnn05GfxEK77jRHjc',
-                    onSave: simple_save_callback
+                    onSave: simple_save_callback,
+                    // selectPanoMode: true,
```

Comment:
We don't have any documentation so i left this line as a reference how to turn on selecting mode. I agree that we shouldn't leave commented code. Let's create new task for write down all constructor options and then remove this line. What do You think?